docs/src/content/docs/getting-started.mdx

```diff
@@ -6,7 +6,7 @@ description: Learn how to start building your next documentation site with Starl
 import { Tabs, TabItem } from '@astrojs/starlight/components';
 
 :::caution[Work in progress]
-Starlight is in early development.
+Starlight is in early development. There will be frequent updates, so be sure to [update Starlight regularly](#updating-starlight)!
```

Comment:
This caution gets pretty massive on mobile, so I’d prefer not to overload it so that people aren’t frightened away.

<img width="320" alt="Starlight getting started page in a mobile viewport width" src="https://github.com/withastro/starlight/assets/357379/28145548-93c6-4786-a23b-66f3ea770985">


```suggestion
Starlight is in early development.
```

I do like the idea though, maybe we just mention something like this in the updating section?